configs/config.toml.SAMPLE

```diff
@@ -54,7 +54,17 @@
 [Storage]
   DataDir = "/my/secure/location"
 
+# Uncomment to enable the metrics module which collects and exposes information
+# useful for external monitoring tools. The port on which the `/metrics`
+# endpoint will be available and the frequency with which the metrics will be
+# collected can be customized using the below parameters.
 # [Metrics]
     # Port = 8080
     # NetworkMetricsTick = 60
     # EthereumMetricsTick = 600
+
+# Uncomment to enable the diagnostics module which exposes information
+# useful for debugging. The port on which the `/diagnostics` endpoint
+# will be available can be customized.
```

Comment:
Let's list what diagnostic information is available and mention this endpoint is not only for debuging but also for diagnostic the current client's status. My proposition:
```
# Uncomment to enable the diagnostics module which exposes information useful
# for debugging and diagnostic client's status. 
#
# Diagnostics module exposes the following information:
# - list of connected peers along with their network id and ethereum operator address
# - information about the client's network id and ethereum operator address 
#
# The port on which the `/diagnostics` endpoint ewill be available can be customized
# below.
```